internal/connect/hwinfo.go

```diff
@@ -41,6 +45,13 @@ func getHwinfo() (hwinfo, error) {
 	}
 	hw.Hostname = hostname()
 	hw.CloudProvider = cloudProvider()
+	hw.MemTotal = systemMemory()
+
+	// Include memory information if possible.
+	mem := systemMemory()
+	if mem > 0 {
+		hw.MemTotal = systemMemory()
+	}
 
```

Comment:
I don't understand L51-54.  L48 has already set `hw.MemTotal`

Reply:
Sorry about that, fixed!